src/CMakeLists.txt

```diff
@@ -194,6 +194,66 @@ endif()
 #set(rdkafka_SRC_DIR ${PROJECT_SOURCE_DIR}
 #    CACHE INTERNAL "${PROJECT_NAME} source dir" FORCE)
 
+# Generate pkg-config file
+set(PKG_CONFIG_NAME
+    "librdkafka"
+)
+set(PKG_CONFIG_DESCRIPTION
+    "The Apache Kafka C/C++ library"
+)
+set(PKG_CONFIG_VERSION
+    "${PROJECT_VERSION}"
+)
+set(PKG_CONFIG_REQUIRES "")
+if(WITH_ZLIB)
+  string(APPEND PKG_CONFIG_REQUIRES "zlib ")
+endif()
+if(WITH_SSL)
+    string(APPEND PKG_CONFIG_REQUIRES "libssl ")
+endif()
+if(WITH_SASL_CYRUS)
+  string(APPEND PKG_CONFIG_REQUIRES "libsasl2 ")
+endif()
```

Comment:
+WITH_ZSTD and +WITH_LZ4_EXT

Reply:
Linking with external LZ4 library (`WITH_LZ4_EXT`) is not yet supported by the CMake build files.
Adding support for that should be its own issue IMHO.